src/CalcViewModel/Common/Utils.h

```diff
@@ -42,10 +42,16 @@
         }\
     } private: t m_##n; public:
 
-#define NAMED_OBSERVABLE_PROPERTY_RW(t, n)\
+#define OBSERVABLE_NAMED_PROPERTY_R(t, n)\
+    OBSERVABLE_PROPERTY_R(t, n)\
+    internal: static property Platform::String^ n##PropertyName {\
+    Platform::String^ get() { return Platform::StringReference(L#n); }\
+    } public:
+
+#define OBSERVABLE_NAMED_PROPERTY_RW(t, n)\
```

Comment:
Does this need a set()?

Reply:
No, and that's intentional.  If it had a setter, then outside clients could change the value of the string returned for the named property.  The property name should be read-only data.